tests/std/tests/P0218R1_filesystem/test.cpp

```diff
@@ -2855,11 +2871,13 @@ void test_status() {
     error_code ec;
     create_file_containing(testFile, L"Hello");
 
-    EXPECT(status(nonexistentPath).type() == file_type::not_found); // should not throw
-    EXPECT(status(nonexistentPath, ec).type() == file_type::not_found);
-    EXPECT(ec.category() == system_category());
-    EXPECT(ec.value() == 2 || ec.value() == 3);
-    EXPECT(ec == errc::no_such_file_or_directory);
+    for (auto&& nonexistent : nonexistentPaths) {
+        EXPECT(status(nonexistent).type() == file_type::not_found); // should not throw
+        EXPECT(status(nonexistent, ec).type() == file_type::not_found);
+        EXPECT(ec.category() == system_category());
+        EXPECT(ec.value() == 2 || ec.value() == 3 || ec.value() == 53);

```

Comment:
Pre-existing, but bigger now: should there be a comment as to what these magic values mean?